include/sway/config.h

```diff
@@ -110,12 +110,19 @@ struct bar_config {
 	 * Always visible in dock mode. Visible only when modifier key is held in hide mode.
 	 * Never visible in invisible mode.
 	 */
-	char *mode;
+	char *display_mode;
```

Comment:
Sorry, nevermind. In the compositor it's fine to keep this a string. We'll want an enum on the swaybar side, though.